spec/relayer/ics-018-relayer-algorithms/README.md

```diff
@@ -266,6 +266,723 @@ The relay process must have access to accounts on both chains with sufficient ba
 
 Any number of relayer processes may be safely run in parallel (and indeed, it is expected that separate relayers will serve separate subsets of the interchain). However, they may consume unnecessary fees if they submit the same proof multiple times, so some minimal coordination may be ideal (such as assigning particular relayers to particular packets or scanning mempools for pending transactions).
 
+### Events
+
+As specified in [ICS 24](../../core/ics-024-host-requirements/README.md#event-logging-system),
+the host state machine MUST provide an event logging system that can make event logs available 
+to relayers (by streaming and/or querying). This section specifies the events emitted by code
+IBC that relayers need to use to complete client, connection and channel lifecycles, relay and 
+timeout packets.
+
+Events emitted by core IBC are JSON-encoded objects following the schema:
```

Comment:
Why are events JSON-encoded, rather than proto-encoded like other IBC data?

Reply:
they will eventually be proto-encoded is my guess: https://github.com/cosmos/ibc-go/issues/54